ajax/libs/title-notify/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "title-notify",
+  "filename": "iNotify.min.js",
+  "version": "1.0.14",
+  "description": "JS achieve the browser title flashing , scrolling, voice prompts , chrome notice.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/jaywcjlove/iNotify.git"
+  },
+  "keywords": [
+    "inotify",
+    "notify",
+    "alert",
+    "notice",
+    "front-end",
+    "title",
+    "notification"
+  ],
+  "author": "kenny wang jaywcjlove <398188662@qq.com>",
+  "homepage": "http://jaywcjlove.github.io/iNotify",
```

Comment:
Let's use http://wangchujiang.com/iNotify/ as the homepage because the like you written always redirect to http://wangchujiang.com/iNotify/.
Thanks.